app/controllers/posts_controller.rb

```diff
@@ -3,10 +3,32 @@ class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
     @posts = Post.where(author_id: params[:user_id]).order(id: :asc)
-    @posts = @posts.paginate(page: params[:page], per_page: 2)
+    @posts = @posts.paginate(page: params[:page], per_page: 10)
   end
 
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
- [ ] Let us avoid repetition. Kindly make use of the strong parameters you defined below, instead of repeating yourself.

  ```
  Post.new(post_params)
  ```